src/styles/container.scss

```
@@ -0,0 +1,9 @@
+.container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin: 50px auto 0;
+  padding: 0 120px;
+  box-sizing: content-box;
```

content-box is the default value of box-sizing. Also, it is better to use border-box to make calculations easier. 
```suggestion
  box-sizing: border-box;
```